packages/rocketchat-message-pin/server/pinMessage.js

```diff
@@ -0,0 +1,86 @@
+Meteor.methods({
+	pinMessage(message, pinnedAt) {
+		if (!Meteor.userId()) {
+			throw new Meteor.Error('error-invalid-user', 'Invalid user', {
+				method: 'pinMessage'
+			});
+		}
+		if (!RocketChat.settings.get('Message_AllowPinning')) {
+			throw new Meteor.Error('error-action-not-allowed', 'Message pinning not allowed', {
+				method: 'pinMessage',
+				action: 'Message_pinning'
+			});
+		}
+		const room = RocketChat.models.Rooms.findOneById(message.rid);
+		if (Array.isArray(room.usernames) && room.usernames.indexOf(Meteor.user().username) === -1) {
+			return false;
+		}
+		let originalMessage = RocketChat.models.Messages.findOneById(message._id);
+		if (!(originalMessage && originalMessage._id)) {
```

Comment:
`if (originalMessage == null || originalMessage._id == null) {` is more readable